Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -730,26 +730,27 @@ def create_issue_command():
 def edit_issue_command(issue_id, mirroring=False, headers=None, status=None, transition=None, **kwargs):
     url = f'rest/api/latest/issue/{issue_id}/'
     issue = get_issue_fields(mirroring=mirroring, **kwargs)
-    jira_req('PUT', url, json.dumps(issue))
+    #jira_req('PUT', url, json.dumps(issue))
```

Comment:
- What will be the flow here if no `status` and no `transition` but another field was changed 
it's look like the change will no be saved
- please remove the comment line & the line of `url` definition if no more needed

Reply:
Without the patch, I will get the following error when trying to close a ticket and set a resolution.

![2022-06-21_16-12-50](https://user-images.githubusercontent.com/2354407/174911636-15eb81b1-ba29-42e9-944f-209a8fba8da6.png)

The reason is because, according to the API guide at https://docs.atlassian.com/software/jira/docs/api/REST/8.22.3/#issue-doTransition, you can only set the resolution field when you do a transition (which is POST). Since the Jira module is setting the field with a PUT operation before the POST operation is done, I effectively can't update the resolution field.
The modification takes care of that by including the fields in the issueJson into the PUT request.